app/indexers/hyrax/active_fedora_file_set_indexer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+module Hyrax
+  class ActiveFedoraFileSetIndexer < ActiveFedora::IndexingService
+    include Hyrax::IndexesThumbnails
+    include Hyrax::IndexesBasicMetadata
+    STORED_LONG = ActiveFedora::Indexing::Descriptor.new(:long, :stored)
+
+    def generate_solr_document # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
This is the most problematic change because it changes the public API.  This change moves `Hyrax::FileSetIndexer` to `Hyrax::ActiveFedoraFileSetIndexer`.  This is needed because following naming conventions, `Hyrax::FileSet` (valkyrie resource) expects its indexer to be named `Hyrax::FileSetIndexer`.

This is a straight move and rename of the class.  The code is unchanged.

Reply:
ah yeah. my other comment. i think i'd rather leave the old class exactly as it was, and introduce special resolution logic for this case.

we could always realign the names later, after the old class has been deprecated and removed.